ui/src/components/messages/Message.tsx

```
@@ -7,23 +7,40 @@ export type MessageProps = {
   ) => {
+    let backgroundColor = 'bg-white dark:bg-dark-bg2'
+    if (negative) backgroundColor = 'bg-red-100 dark:bg-red-900'
+    if (positive) backgroundColor = 'bg-green-100 dark:bg-green-900'
+
     return (
       <div
         ref={ref}
-        className="bg-white dark:bg-dark-bg2 shadow-md border rounded p-2 h-[84px] relative"
+        className={`${backgroundColor} shadow-md border rounded p-2 relative`}
       >
         <button onClick={onDismiss} className="absolute top-3 right-2">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)